<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScopeWindow</class>
 <widget class="QWidget" name="ScopeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>665</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Scope</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/resources.qrc">
    <normaloff>:/icons/scope.png</normaloff>:/icons/scope.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="ScopeWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Scope</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btnStop">
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnStart">
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Timebase</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDial" name="dlTimebase"/>
          </item>
          <item>
           <widget class="QLabel" name="lbTimebase">
            <property name="text">
             <string>1000 ms/div</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Trigger</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Trigger Level:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QSpinBox" name="sbTriggerLevel"/>
             </item>
             <item row="0" column="0" colspan="3">
              <widget class="QComboBox" name="cbTriggerMode">
               <item>
                <property name="text">
                 <string>Free Run</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Rising Edge</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Falling Edge</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Trigger Delay:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QSpinBox" name="sbTriggerDelay">
               <property name="suffix">
                <string>ms</string>
               </property>
               <property name="maximum">
                <number>3000</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Channels</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <row>
             <property name="text">
              <string>New Row</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>New Row</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>New Row</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Universe</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Enabled</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Colour</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Trigger</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>16-Bit</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="0" column="2">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="0" column="3">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="1" column="2">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="1" column="3">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="2" column="2">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="2" column="3">
             <property name="text">
              <string>1</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btnAddChannel">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRemoveChannel">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScopeWidget</class>
   <extends>QWidget</extends>
   <header>scopewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
